pySDC/projects/DAE/tests/test_MPI_sweepers.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+
+@pytest.mark.mpi4py
+@pytest.mark.parametrize('num_procs', [2, 3])
+def testOrder(num_procs):
+    r"""
+    Test checks if order of accuracy is reached for the MPI sweepers.
+    """
+
+    import os
+    import subprocess
+
+    # Set python path once
+    my_env = os.environ.copy()
+    my_env['PYTHONPATH'] = '../../..:.'
+    my_env['COVERAGE_PROCESS_START'] = 'pyproject.toml'
+    cwd = '.'
+
+    cmd = ('mpirun -np ' + str(num_procs) + ' python pySDC/projects/DAE/run/accuracy_check_MPI.py').split()
```

Comment:
Sorry for the comment with the environment. Clearly, you did this correctly already ;)

There is a simple solution to the path stuff. You can import the file you need and then get the path with `.__path__`. I do that [here](https://github.com/Parallel-in-Time/pySDC/blob/e372a43c621e1bb8ef86c68e02393a3338c096d8/pySDC/projects/Resilience/tests/test_vdp.py#L18) for instance.
I am wondering, though. Why do the tests pass if this path is incorrect?

Reply:
I guess it is something local that does not work for me. Maybe something stupid.. I tried now your path solution. Then I get a ``ModuleNotFoundError: No module named 'pySDC'``

Reply:
What python is used when you do `mpiexec`? Can you run `mpiexec which pytest` and check that it's the same as simply `which pytest`? Are you using conda/mamba? Do other scripts work? 
If some imports work, but not others, maybe you installed pySDC with pip but not in editable mode? Maybe try `mpirun pip install -e <path/to/pySDC>`.

Reply:
I made it already work ..somehow. This was interesting when I comment ``my_env['PYTHONPATH'] = '../../..:.'`` then it does work (locally). The tests then pass. So for pushing I added this again since I guess this is needed for the CI pipeline.

Reply:
I recommend to do `pip install -e` instead of adding pySDC to python path because then you don't need to change your python path in your bashrc. I assume this will also work locally then. Of course, you install to a specific python version. So this may not be what you want and you can safely ignore this comment.